app/cdap/components/common/ButtonWidget/index.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Button } from '@material-ui/core';
+import { blue, grey } from '@material-ui/core/colors';
+import styled from 'styled-components';
+
+export const AddTransformationButton = styled(Button)`
```

Comment:
I don't understand the naming of this - 

the folder is called buttonWidget but the AddTransformationBuitton is the one being exported? furthermore this doesn't use any jsx so it should be a ts file. Also why are you adding this common  folder to the components? we do not need a common folder in there because they're all shared components.

Reply:
renamed the button to StyledButton and changed extension of the file .ts

Reply:
@seanfeld plz let me know if we can move these components to a shared folder in components/ or if I should create a new folder in component/ called WranglerComponents. Based on your inputs I will move the components from common folder and delete the folder.

Reply:
I'd prefer you name it WranglerV2 or something like that - its already in components/ so you don't need to repeat components.